src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,265 @@
+        } else {
+            throw new DukeException("Illegal Todo Prompt detected.");
+        }
+    }
+
+    private static void printAddSuccessMessage(Task task) {
+        System.out.print(DIVLINE);
+        System.out.println("\tGot it. I've added this task:\n" +
+                "\t  " + task.toString());
+        System.out.println("\tNow you have " + taskList.size() + " tasks in the list.");
+        System.out.print(DIVLINE);
+    }
+
+    private static void printDeleteSuccessMessage(Task task) {
+        System.out.print(DIVLINE);
+        System.out.println("\tNoted. I've removed this task: \n" +
+                "\t  " + task.toString());
+        System.out.println("\tNow you have " + taskList.size() + " tasks in the list.");
+        System.out.print(DIVLINE);
+    }
+
+    private static void completeTask(int taskIndex) throws DukeException {
+        if (taskList.get(taskIndex).getIsDone()){
+            throw new DukeException("Task has already been marked done.");
+        }
+        taskList.get(taskIndex).markAsDone();
+        System.out.print(DIVLINE);
+        System.out.println("\tNice! I've marked this task as done: \n" +
+                "\t" + taskList.get(taskIndex).toString());
+        System.out.print(DIVLINE);
+    }
+
+    private static void deleteFromList(int taskIndex) throws DukeException, IOException {
+        if (taskIndex >= taskList.size()) {
+            throw new DukeException("Task does not exist");
+        }
+        Task toBeDeleted = taskList.get(taskIndex);
+        // Some manipulation here
+        taskList.remove(taskIndex);
+        writeToFile(initializeTextToWrite());
+        printDeleteSuccessMessage(toBeDeleted);
+    }
+
+    public static void main(String[] args) throws IOException {
+        createFileIfNotExist();
+        ArrayList<String> textInput = readFile();
+        initializeTaskList(textInput);
+        System.out.print(DIVLINE + GREETINGS + DIVLINE);
+        Scanner in = new Scanner(System.in);
+        while (!isExiting) {
+            try{
+                String prompt = in.nextLine();
+                processPrompt(prompt);
+            } catch (DukeException e) {
+                System.out.println(DIVLINE + e.getIllegalKeywordMessage());
+                System.out.print(DIVLINE);
+            }
+        }
+        // writeToFile(initializeTextToWrite());
+        System.out.println(DIVLINE + PARTINGS + DIVLINE);
+    }
+
+    private static String initializeTextToWrite() {
+        String text = "";
+        for (int i=0; i<taskList.size(); i++) {
+            text = text + taskList.get(i).toText(i+1)+ System.lineSeparator();
+        }
+        return text;
+    }
+
+    private static void initializeTaskList(ArrayList<String> fileInput) {
+        for (int i=0; i<fileInput.size(); i++) {
+            //create tasklist according to file storage
+            String[] arr = fileInput.get(i).split("\\|");
+            switch (arr[0]) {
+                case "T":
```

Comment:
Using a single alphabets to describe cases does not provide greater clarity on what cases the method is handling. You could see [Using names to explain](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#use-name-to-explain) for alternatives.